pages/login.tsx

```diff
@@ -122,17 +111,16 @@ export const getServerSideProps: GetServerSideProps = async (
   ctx: GetServerSidePropsContext
 ) => {
   const userCredentials = await ProtectedRoute(ctx);
-  if (userCredentials.props.token) {
-    return {
-      redirect: {
-        permanent: false,
-        destination: '/uploader',
-      },
-      props: {},
-    };
-  } else {
+  if (!userCredentials.props.token) {
     return { props: {} };
   }
+  return {
+    redirect: {
+      permanent: false,
+      destination: '/uploader',
```

Comment:
we should make this the path of the protected route the took them to the login screen or the page that they came from if they clicked the login button manually